hknweb/events/templates/events/show_details.html

```diff
@@ -81,7 +83,7 @@ <h3 class="rsvp-list-title">RSVPs</h3>
         <p class="rsvp-list"> Count: {{ rsvps.count }} </p>
       {% endif %}
       {% for rsvp in rsvps %}
-        <p class="rsvp-list">{{ rsvp.user.first_name }} {{ rsvp.user.last_name }} ({{ rsvp.user }})</p>
+        <p class="rsvp-list">{{ rsvp.user.first_name }} {{ rsvp.user.last_name }} ({{ rsvp.user.username }})</p>
```

Comment:
I think I already fixed this in the earlier pull request. Did you pull from master?

Reply:
Haven't pulled, but this should be fine because it is the exact same change